lib/rpm_package.py

```diff
@@ -151,7 +151,7 @@ def _load(self):
 
             # load distro files
             files = self.package_data.get('files', {}).get(
-                distro_attrib_name, {}).get(self.distro.version, {})
+                distro_attrib_name, {}).get(self.distro.version, {}) or {}
```

Comment:
OH> I don't get where this is needed. 

In the case where we have files/CentOS/7 attributes defined, but nothing below them. In this case, files would be assigned None and files.get('build_files') right below this line would fail.

This happens in some yaml files (eg qemu) after PR versions/#89. We could delete the 'files' attrib in this case in that PR, but nevertheless this change is welcome IMHO.